### PR TITLE
add html-css-popup code for review

### PR DESCRIPTION
**Pull request by DarlaMoonlight**
### HTML & CSS practice: Hooli-style Popup

[Demo](https://darlamoonlight.github.io/kottans_html_css_popup/)
[Code base](https://github.com/DarlaMoonlight/kottans_html_css_popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/DarlaMoonlight/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (lenadgit) [inline submissions/DarlaMoonlight/html-css-popup/index.html:5]: Metatag Viewport is mandatory for mobile markup.
https://serpstat.com/ru/blog/chto-takoe-metateg-viewport/
Could you please add it?
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`

File: submissions/DarlaMoonlight/html-css-popup/style.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+.popup-list__item-link{
+	display: grid;
+	grid-template-columns: 1fr;
+	justify-items: center;
+	color: rgb(255, 255, 255);
+	padding:10px 5px;
+	font-size: 0.8em;  
+}
+.navigation-list__item-link{
```
Comment (lenadgit) [inline submissions/DarlaMoonlight/html-css-popup/style.css:142]: could you please remove unused class?

File: submissions/DarlaMoonlight/html-css-popup/style.css [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+  	align-items: center;
+}
+
+.navigation-list__item {
+	display: inline-block;
+	padding: 10px;
+}
+
+.navigation-list__item  .list-item_img {
+
```
Comment (lenadgit) [inline submissions/DarlaMoonlight/html-css-popup/style.css:79]: could you please remove an empty line at this class?

File: submissions/DarlaMoonlight/html-css-popup/style.css [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+	display: inline;
+}
+
+.more_check:checked ~label > .more-button__more{
+	display: none;
+}
+.user-icon
+{
+	border-radius: 50%;
+}
```
Comment (lenadgit) [inline submissions/DarlaMoonlight/html-css-popup/style.css:206]: could you please format this file by autoformatting? (to add alignments, to align brackets in the right order)

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Comment by lenadgit** [comment]
Excellent job! really cool.
Thank you for the submission. Feel free taking the next course module while you submission is being reviewed

File: submissions/DarlaMoonlight/html-css-popup/style.css (hunk 4 above)
Comment (DarlaMoonlight) [reply submissions/DarlaMoonlight/html-css-popup/style.css:201]: Hi! Thanks for the review. I corrected my code. Please, check it.

File: submissions/DarlaMoonlight/html-css-popup/index.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+                                <a href="#" class="popup-list__item-link"><img src="img/videos.png" alt="Videos" class="list-item_img">
+                                    <span>Videos</span></a>
+                            </li>
+                            <li class="popup-list__item">
+                                <a href="#" class="popup-list__item-link"><img src="img/search.png" alt="Search" class="list-item_img">
+                                    <span>Search</span></a>
+                            </li>
+                        </ul>
+                        <input type="checkbox" id="more_option" class="visuallyhidden more_check">
+                        <label for="more_option" class="more-button">
```
Comment (A-Ostrovnyy) [inline submissions/DarlaMoonlight/html-css-popup/index.html:67]: Do you really need here `display: grid` for centering one word?

**Review by A-Ostrovnyy — APPROVED** [review]
Really good work!
@lenadgit your turn

**Review by lenadgit — APPROVED** [review]

**Pull request merged**